futaba/sql/models/roles.py

```diff
@@ -169,11 +172,68 @@ def add_pingable_role_channel(self, guild, channel, role):
         assert guild == channel.guild
 
         ins = self.tb_pingable_role_channel.insert().values(
-            guild_id=guild.id, channel_id=channel.id, role_id=role.id
+            guild_id=guild.id, channel_id=channel.id, role_id=role.id, original_role_id=original.id
```

Comment:
What if `original` is `None`?

Reply:
Then it gets inserted as null in the table. It's a nullable type.

Reply:
Yeah but if `original` is None` then `original.id` will raise an `AttributeError`. You'll need something like `getattr(original, 'id', None)` to safely extract `id` if it exists else `None`.